docpipeline/generate.py

```diff
@@ -375,14 +375,25 @@ def build_new_docs(bucket_name, credentials):
     client = storage_client(credentials)
     all_blobs = list(client.list_blobs(bucket_name))
     docfx_blobs = [blob for blob in all_blobs if blob.name.startswith(DOCFX_PREFIX)]
-    other_blobs = [blob for blob in all_blobs if not blob.name.startswith(DOCFX_PREFIX)]
-    other_names = set(map(lambda b: b.name, other_blobs))
+    other_blobs = {b.name: b for b in all_blobs if not b.name.startswith(DOCFX_PREFIX)}
+    other_names = set(other_blobs.keys())
```

Comment:
No need for `other_names`. We can use `other_blobs` in place with no changes since it's a dict.

Reply:
Sounds good, done!